source/Cute/Commands/Server/ServerSechedulerCommand.cs

```diff
@@ -1,16 +1,231 @@
+
+    private Settings? _settings;
+
+    private readonly ILogger<Scheduler> _cronLogger = cronLogger;
+
+    private readonly Scheduler _scheduler = new Scheduler(cronLogger,
+            new SchedulerOptions
+            {
+                DateTimeKind = DateTimeKind.Utc
+            }
+        );
+
+    private Dictionary<Guid, CuteContentSyncApi> _cronTasks = [];
+
+    public override void ConfigureWebApplication(WebApplication webApp)
+    {
+        webApp.MapPost("/reload", RefreshSchedule).DisableAntiforgery();
+    }
+
+    public override void ConfigureWebApplicationBuilder(WebApplicationBuilder webBuilder)
+    {
+        webBuilder.WebHost.ConfigureKestrel(web =>
+        {
+            web.ListenLocalhost(_settings!.Port);
+        });
+    }
+
+    public override async Task RenderHomePageBody(HttpContext context)
+    {
+        await context.Response.WriteAsync($"<h4>Scheduled Tasks</h4>");
+
+        await context.Response.WriteAsync($"<table>");
+        await context.Response.WriteAsync($"<tr>");
+        await context.Response.WriteAsync($"<th style='width:20%'>Task</th>");
+        await context.Response.WriteAsync($"<th>Schedule</th>");
+        await context.Response.WriteAsync($"<th style='width:13%'>Cron</th>");
+        await context.Response.WriteAsync($"<th style='width:23%'>Next run</th>");
+        await context.Response.WriteAsync($"</tr>");
+
+        var nextRun = _scheduler.GetNextOccurrences()
+            .SelectMany(i => i.ScheduledTasks, (i, j) => new { j.Id, i.NextOccurrence })
+            .ToDictionary(o => o.Id, o => o.NextOccurrence);
+
+        foreach (var (key, entry) in _cronTasks)
+        {
+            await context.Response.WriteAsync($"<tr>");
+            await context.Response.WriteAsync($"<td>{entry.Key}</td>");
+            await context.Response.WriteAsync($"<td>{entry.Schedule}</td>");
+            await context.Response.WriteAsync($"<td>{entry.Schedule?.ToCronExpression()}</td>");
+            await context.Response.WriteAsync($"<td>");
+            await context.Response.WriteAsync($"{nextRun[key]:R}<br>");
+            await context.Response.WriteAsync($"</td>");
+            await context.Response.WriteAsync($"</tr>");
+        }
+
+        await context.Response.WriteAsync($"</table>");
+
+        await context.Response.WriteAsync($"<form action='/reload' method='POST' enctype='multipart/form-data'>");
+        await context.Response.WriteAsync($"<input type='hidden' name='command' value='reload'>");
+        await context.Response.WriteAsync($"<button type='submit' style='width:100%'>Reload schedule from Contentful</button>");
+        await context.Response.WriteAsync($"</form>");
+    }
+
+    // ...
+
+    public override async Task<int> ExecuteCommandAsync(CommandContext context, Settings settings)
+    {
+        _settings = settings;
+
+        LoadSyncApiEntries();
+
+        RefreshScheduler();
+
+        if (Environment.OSVersion.Platform == PlatformID.Win32NT)
```

Comment:
Remove this section. we don't want the server to launch a browser, and don't want windows specific code if at all possible